tests/test_human.py

```diff
@@ -0,0 +1,176 @@
+#
+# Copyright 2018 Sveta Haas.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Refer to the README and COPYING files for full details of the license
+#
+from __future__ import print_function
+
+import human
+import fixtures
+import pytest
+from human import Human
+
+
+
+@pytest.fixture
```

Comment:
1. If you intend this fixture to have a module scope (setup once and teardown when all tests that required it finished).
The syntax is this: @pytest.fixture(scope='module')
2. No one is "requiring" this fixture in any of your tests, so it will never run.